bindings/electron/README.md

```diff
@@ -1,9 +1,41 @@
 # libparsec-electron-bindings
 
+## Build
+
```

Comment:
All that in the current directory right ?

Reply:
For the `npm` command yes